workshop 01 - Introduction to Quantum Computing/intro-to-qsharp.ipynb

```diff
@@ -5,7 +5,8 @@
    "metadata": {},
```

Comment:
[A note on commit message]

Hey @ricardoprins, instead of explaining what this commit does can you please re-frame it to convey the rationale behind this?

Reply:
Furthermore, prepend the area of work which is `workshop 01` here.